.github/workflows/Package.yml

```diff
@@ -66,24 +66,38 @@ jobs:
 
       - name: 🔨 [build] Build Python package (source distribution)
         if: inputs.requirements == ''
-        run: python -m build --no-isolation --sdist
+        run: python -m build --sdist
 
       - name: 🔨 [build] Build Python package (binary distribution - wheel)
         if: inputs.requirements == ''
+        run: python -m build --wheel
+
+      # build (not isolated)
+
+      - name: 🔧 [build] Install dependencies for packaging and release
+        if: inputs.requirements == 'no-isolation'
+        run: python -m pip install build
+
+      - name: 🔨 [build] Build Python package (source distribution)
+        if: inputs.requirements == 'no-isolation'
+        run: python -m build --no-isolation --sdist
+
+      - name: 🔨 [build] Build Python package (binary distribution - wheel)
+        if: inputs.requirements == 'no-isolation'
         run: python -m build --no-isolation --wheel
 
       # setuptools
 
       - name: 🔧 [setuptools] Install dependencies for packaging and release
-        if: inputs.requirements != ''
+        if: inputs.requirements != '' && inputs.requirements != 'no-isolation'
```

Comment:
Did you test this branch in some other repo? I am unsure about this *if* statement requiring `${{ }}`. In some repo (which I don't remember) I had problems when using two conditions (as in here).

Reply:
As far as I remember, `${{ ... }}` is not needed in `if:`.

It's tested in pyEDAA.Launcher: https://github.com/edaa-org/pyEDAA.Launcher/actions/runs/1917405067
Here the used reference: https://github.com/edaa-org/pyEDAA.Launcher/blob/paebbels/improvements/.github/workflows/Pipeline.yml#L56

Reply:
Also tested in https://github.com/pyTooling/pyTooling/runs/5383059245?check_suite_focus=true.